app/controllers/host_controller.rb

```diff
@@ -116,7 +116,8 @@ def guest_applications
   # Show the main Host list view overriding method from Mixins::GenericListMixin
   def show_list
     session[:host_items] = nil
-    process_show_list
+    options = {:no_checkboxes => ActiveRecord::Type::Boolean.new.cast(params[:no_checkboxes])}
```

Comment:
Why that `ActiveRecord::Type::Boolean.new()` here?

Reply:
Because it could be `"true"`, `"false"` string or nil. BZ is only for display running VM, but there is also an option to display running host from CU chart menu for Cluster.